.travis.yml

```diff
@@ -4,5 +4,6 @@ go:
   - 1.6
 
 script:
-  - go build
-  - go test ./...
+  - make init
```

Comment:
That will make build a lot slower, because there is a lot of stuff to pull. We could run the tests also with just `go test ./...`, removing the necessity for additional dependencies. What do you think?